modules/ROOT/pages/ps-config-log-forwarding.adoc

```diff
@@ -0,0 +1,360 @@
+= Configuring Logging for a Private Space 
```

Comment:
This page is not needed anymore. External log forwarding will be replaced with custom log4j support, and there is nothing to configure for Anypoint log forwarding as it is always available.